service/lib/agama/ipmi.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2022-2023] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "agama/config"
+require "tempfile"
+
+module Agama
+  # Class for very basic IPMI support
+  #
+  # Use for reporting current installer state to IPMI.
+  #
+  # Implemented:
+  #
+  # TBD:
+  # * STARTED
+  # * WAITING (?)
+  # * ABORTED
+  # * FAILED
+  # * FINISHED
```

Comment:
is comment correct?

Reply:
I believe I've updated that ... may be some hanging update ... will check